scripts/build-linux.sh

```diff
@@ -258,7 +265,11 @@ if [ ${CPU} = vexriscv ] && [ ${BUILD_BUILDROOT:-0} = 1 ]; then
 		dtc -I dts -O dtb -o $TARGET_LINUX_BUILD_DIR/rv32.dtb $TARGET_LINUX_BUILD_DIR/rv32.dts
 
 		cd $BD_SRC
-		make BR2_EXTERNAL=$LLV_SRC/buildroot/ litex_vexriscv_defconfig
+                if [ "x$BR2_EXTERNAL" = "x" ]; then
```

Comment:
there is something wrong with the indent + I guess `""` is not needed if you use the x trick

Reply:
Thanks, I'll fix it.  My editor showed it correctly, it probably has a different tab setting or something.